airflow/Dockerfile

```diff
@@ -12,6 +15,8 @@ RUN set -ex \
     && useradd -s /bin/bash -d ${AIRFLOW_HOME} airflow \
     && pip install -U pipenv
 
+COPY --from=wheels /opt/wheels /opt/wheels
```

Comment:
this makes a change in the wheels trigger a new `pipenv install`. Is it meant it because `pipenv install` needs the wheels to work right?
(could still be moved right after `COPY Pipfile.lock .` even if that's the case)

Reply:
Not true, `.docker/entrypoint.sh` installs the wheels. But it shouldn't be necessary to do this at runtime if it can be done at build time. Move this right after `pipenv install` and add the additional installs that are in the entry point?